app/src/main/dist/config/crate.yml

```diff
@@ -88,8 +88,16 @@ auth.host_based.enabled: true
 #   Note that this authentication method is only available via Postgres
 #   Protocol, meaning that with this configuration, connecting from non-local
 #   hosts via HTTP would not be possible!
+
 auth:
   host_based:
+    # Node level JWT defaults which are only used if JWT is used as
```

Comment:
I think we once had a discussion if we continue to add documentation to our default config file. This was raised as the content inside this file got outdated. I think the outcome was not to continue doing so, not sure, but I'd vote for getting rid of all documentation here...

Reply:
removed docs but kept example values, similar to existing entries, for example
```
#  trust:
#    http_default_user: dustin
```